podvm-mkosi/Makefile

```diff
@@ -12,16 +12,23 @@ fedora-binaries-builder:
 	@echo "Building fedora-binaries-builder image..."
 	docker buildx build \
 		-t fedora-binaries-builder \
+		--build-arg CAA_SRC=https://github.com/katexochen/cloud-api-adaptor \
+		--build-arg CAA_SRC_REF=feat/mkosi-dmverity-nix \
+		--no-cache \
```

Comment:
We could parameterize those in the the Makefile or just drop before merge. I opt for the latter to reduce the noise

Reply:
Yes, that commit should be dropped.